pyre/extra_tests/snippets/attribute_error_lookup_context.py

```
@@ -0,0 +1,42 @@
+# PyPy 9883bb2a9d `enrich_attribute_error`, with CPython 3.14's distinction
+# between an omitted slot and an explicitly supplied None.  An inner lookup's
+# more specific context wins over the outer `__getattr__` receiver, and getattr
+# keeps the caller's name object.
+
+
+class Empty:
+    def __getattr__(self, name):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__getattr__`

(ANN204)